### PR TITLE
provide a default for options

### PR DESCRIPTION
**Pull request by thisconnect**
It seems new IPFS() isn't working at the moment, see this PR https://github.com/ipfs/js-ipfs/pull/811/files

cc https://github.com/ipfs/js-ipfs/pull/811

### CODE REVIEW

**Comment by thisconnect** [comment]
that didn't really work in node v4 :(

File: src/core/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const boot = require('./boot')
 const components = require('./components')
 
 class IPFS extends EventEmitter {
-  constructor (options) {
+  constructor (options = {}) {
```
Comment (dignifiedquire) [inline src/core/index.js]: you can just do `options = options != null ? options : {}`

**Comment by VictorBjelkholm** [comment]
We should add a test case to make sure all defaults are working as well, so this doesn't bite us again.

**Comment by diasdavid** [comment]
@VictorBjelkholm defaults mean "connect to the real network"

**Comment by VictorBjelkholm** [comment]
@diasdavid we're currently doing that anyways in the tests, for example: https://github.com/ipfs/js-ipfs/blob/master/test/core/create-node.spec.js

Should just have a additional test-case with no options provided.

**Comment by diasdavid** [comment]
We don't interfere with user repo in tests.

**Pull request merged**